diff --git a/src/schema.rs b/src/schema.rs
index 9a96457..4393853 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -8,14 +8,14 @@ diesel::table! {
         cover_url_small -> Nullable<Text>,
         mbid -> Nullable<Text>,
         musicbrainz_title -> Nullable<Text>,
         musicbrainz_artist -> Nullable<Text>,
         musicbrainz_length -> Nullable<Int4>,
         mistag_lock -> Bool,
-        aliases_artist -> Nullable<Array<Nullable<Text>>>,
-        aliases_title -> Nullable<Array<Nullable<Text>>>,
+        aliases_artist -> Nullable<Array<Text>>,
+        aliases_title -> Nullable<Array<Text>>,
     }
 }
 
 diesel::table! {
     players (id) {
         id -> Int4,
@@ -44,17 +44,17 @@ diesel::table! {
         song_id -> Int4,
         player_id -> Int4,
         league -> Int2,
         submitted_at -> Timestamp,
         play_count -> Int4,
         score -> Int4,
-        track_shape -> Array<Nullable<Int4>>,
-        xstats -> Array<Nullable<Int4>>,
+        track_shape -> Array<Int4>,
+        xstats -> Array<Int4>,
         density -> Int4,
         vehicle -> Int2,
-        feats -> Array<Nullable<Text>>,
+        feats -> Array<Text>,
         song_length -> Int4,
         gold_threshold -> Int4,
         iss -> Int4,
         isj -> Int4,
     }
 }
@@ -73,13 +73,13 @@ diesel::table! {
 diesel::table! {
     songs (id) {
         id -> Int4,
         title -> Text,
         artist -> Text,
         created_at -> Timestamp,
-        modifiers -> Nullable<Array<Nullable<Text>>>,
+        modifiers -> Nullable<Array<Text>>,
     }
 }
 
 diesel::joinable!(extra_song_info -> songs (song_id));
 diesel::joinable!(scores -> players (player_id));
 diesel::joinable!(scores -> songs (song_id));
